.github/workflows/benchmark.yml

```
@@ -0,0 +1,26 @@
+name: Run benchmarks
+
+on:
+  pull_request:
+
+jobs:
+  Benchmark:
+    runs-on: ubuntu-latest
+    env:
+      JULIA_DEBUG: BenchmarkCI
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: 1.3
+      - name: Install dependencies
+        run: julia -e 'using Pkg; pkg"add PkgBenchmark https://github.com/tkf/BenchmarkCI.jl"'
+      - name: Run benchmarks
+        run: julia -e "using BenchmarkCI; BenchmarkCI.judge()"
```

It now succeeds at running the benchmark for target https://github.com/JuliaImages/ImageFiltering.jl/runs/403909723#step:5:160 but fails when running it for baseline (`origin/master`). Maybe you can test that it works by:

```suggestion
        run: julia -e "using BenchmarkCI; BenchmarkCI.judge(baseline = "HEAD")"
```

Then revert this patch just before merge.